src/plugins/controls/public/range_slider/components/range_slider_editor_options.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useState } from 'react';
+import { EuiFormRow, EuiFieldNumber } from '@elastic/eui';
+
+import type { RangeSliderEmbeddableInput } from '../../../common/range_slider/types';
+import { ControlEditorProps } from '../../types';
+
+import { RangeSliderStrings } from './range_slider_strings';
+
+interface RangeSliderEditorState {
+  step: number;
+}
+
+export const RangeSliderEditorOptions = ({
+  initialInput,
+  onChange,
+  setControlEditorValid,
+}: ControlEditorProps<RangeSliderEmbeddableInput>) => {
+  const [state, setState] = useState<RangeSliderEditorState>({
+    step: initialInput?.step || 1,
```

Comment:
```suggestion
    step: initialInput?.step ?? 1,
```

Reply:
I _think_ the `||` is actually valid here as zero is an invalid step and `||` will fall back when the `initialInput.step` is 0 but `??` won't.

Reply:
Ah! Totally fair 🖖 good call